apps/src/blockly/addons/contextMenu.js

```diff
@@ -236,4 +308,14 @@ function hasShadowChildren(block) {
   return shadowChildCount(block) > 0;
 }
 
+function isCurrentTheme(theme, workspace) {
+  return (
+    workspace?.getTheme().name === theme ||
+    localStorage.getItem(blocklyTheme) === theme
+  );
+}
+
+function isMusicLabTheme(workspace) {
+  return workspace.getTheme().name === 'musiclabdark';
```

Comment:
another constant here?

Reply:
Sure, I wasn't sure if this one was needed because it's only used once.